src/Symfony/Component/Notifier/Bridge/Slack/SlackOptions.php

```diff
@@ -22,11 +23,15 @@
  */
 final class SlackOptions implements MessageOptionsInterface
 {
+    private const MAX_BLOCKS = 50;
+
     private $options;
 
     public function __construct(array $options = [])
     {
         $this->options = $options;
+
+        $this->validateNumberOfBlocks();
```

Comment:
If I'm not wrong, this will trigger an exception when the use creates an object with 50 elements (which is ok)

Reply:
thanks